R/as_dribble.R

```diff
@@ -34,15 +34,13 @@ as_dribble.dribble <- function(x, ...) x
 #' @export
 #' @rdname as_dribble
 as_dribble.character <- function(x, ...) {
-  ## TO DO: we should accept x with length > 1
-  drive_path(x)
+  do.call(rbind, purrr::map(x, drive_path))
 }
 
 #' @export
 #' @rdname as_dribble
 as_dribble.drive_id <- function(x, ...) {
-  ## TO DO: we should accept x with length > 1
-  drive_get(x)
+  do.call(rbind, purrr::map(x, drive_get))
```

Comment:
Re:, e.g. `drive_path()` not being vectorized but wanting other functions like `drive_delete()` to accept multiple files. We forgot to circle back to this! This should take care of it. Can you think of downstream changes this suggests? At the very least, some tests of basic dribble formation with vector input would be good.

We probably want to use one of purrr's safer mapping strategies, for when there's one bad path or id among others that work? But perhaps we should open an issue for that and  not let that keep us from merging this.